Managers/CardManager.gd

```diff
@@ -9,6 +9,11 @@ signal on_deck_initialized
 var card_container: CardContainer = null
 # We store the current deck in CardManager because we may want to access it outside of battle.
 var current_deck: Array[CardBase] = []
+var default_deck: CardSet = null
+
+
+func _ready():
+	default_deck = load("res://Cards/CardSets/DefaultDeck.tres")
```

Comment:
where was it loaded from before ? from the scene directly ? if so, should the scene file be edited to remove the deck from there ?

Reply:
It was in CardContainer and set in the inspector using an @export. Since this is an AutoLoaded singleton, we can’t use the inspector to set it so that’s why I’m loading it like this. I already removed the default_deck from CardContainer